doc/api/http2.md

```diff
@@ -2723,6 +2723,26 @@ added: v8.4.0
 
 The request method as a string. Read-only. Examples: `'GET'`, `'DELETE'`.
 
+#### request.authority
```

Comment:
Sections are sorted alphabetically, so `#### request.authority` needs to be placed after `#### request.aborted` and `#### request.scheme` after `#### request.rawTrailers`.